pkg/resource/deploy/import.go

```diff
@@ -257,7 +257,7 @@ func (i *importer) registerProviders(ctx context.Context) (map[resource.URN]stri
 		}
 
 		state := resource.NewState(typ, urn, true, false, "", inputs, nil, "", false, false, nil, nil, "", nil, false,
-			nil, nil, nil, "", false, "")
+			nil, nil, nil, "", false, "", nil, nil)
```

Comment:
Again I think here (and the other nil, nil) in this file on 360 should set the timestamps to Now.

Reply:
I have all of the code for setting timestamps in the step.go's *Step.Apply() method complete() return callback for consistency.